hphp/hack/hhi/stdlib/builtins_mysqli.hhi

```diff
@@ -0,0 +1,374 @@
+<?hh // decl /* -*- php -*- */
+class mysqli_sql_exception extends RuntimeException  {
```

Comment:
We don't have that.

https://github.com/facebook/hhvm/blob/eab25645565957b29fafd7d9032c33d599b75df3/hphp/runtime/ext/mysql/ext_mysqli.php#L858